app/api/routers/breeze_buddy/templates/version_handlers.py

```
@@ -0,0 +1,103 @@
+async def rollback_template_handler(
+    template_id: str, body: RollbackTemplateRequest, current_user: UserInfo
+):
+    template = await _load_and_authorize(template_id, current_user, "rollback template")
+    require_admin_or_reseller_owner(
+        current_user, template.reseller_id, operation="rollback template"
+    )
+    try:
+        new_head = await rollback_template_to_version(
+            template_id, body.version, changed_by=current_user.username
+        )
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)